src/load_inputs/load_demand_data.jl

```diff
@@ -1,9 +1,28 @@
+function get_demand_dataframe(path)
+    filename = "Demand_data.csv"
+    deprecated_synonym = "Load_data.csv"
+    df = load_dataframe(path, [filename, deprecated_synonym])
+    # update column names
+    old_columns = find_matrix_columns_in_dataframe(df, DEMAND_COLUMN_PREFIX_DEPRECATED[1:end-1],
+                                     prefixseparator='z')
+    old_column_symbols = Symbol.(DEMAND_COLUMN_PREFIX_DEPRECATED*string(i) for i in old_columns)
+    if length(old_column_symbols) > 0
+        @info "$DEMAND_COLUMN_PREFIX_DEPRECATED is deprecated. Use $DEMAND_COLUMN_PREFIX."
+        new_column_symbols = Symbol.(DEMAND_COLUMN_PREFIX*string(i) for i in old_columns)
+        rename!(df, Dict(old_column_symbols .=> new_column_symbols))
+    end
+    return df
+end
+
+DEMAND_COLUMN_PREFIX = "Demand_MW_z"
+DEMAND_COLUMN_PREFIX_DEPRECATED = "Load_MW_z"
```

Comment:
These  'global constants' should be made immutable by turning them into functions, or some other more Julian way.